controllers/depresolver/depresolver_test.go

```diff
@@ -1283,6 +1283,145 @@ func TestResolveLoggerEmptyValues(t *testing.T) {
 	assert.Equal(t, SimpleFormat, config.Log.Format)
 }
 
+func TestNsServerNamesWithMultipleExtClusterGeoTag(t *testing.T) {
+	// arrange
+	defer cleanup()
+	customConfig := predefinedConfig
+	customConfig.DNSZone = "k8gb-test-preprod.gslb.cloud.example.com"
+	customConfig.EdgeDNSZone = "cloud.example.com"
+	customConfig.ClusterGeoTag = "westus-1"
```

Comment:
is it intentionally formed as non-aws looking like tag?

Reply:
I replaced with standard AWS tags:  https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

Reply:
In this case, it would look like `us-west-1`?

Reply:
@somaritane  [sure](https://github.com/AbsaOSS/k8gb/pull/459/files#diff-384d90ba8267cb88a0aab2845e0778c734551e380ce8040b033b006584153fdeR1292)

Reply:
IMO It's better if we use vendor-neutral tags in the examples and tests, for example, `location-1`, `location-2`